src/queries.ts

```diff
@@ -39,6 +39,7 @@ interface DesktopQueryParams extends BaseQueryParams {
   bid_window: string;
   bid_afk: string;
   filter_afk: boolean;
+  neverTreatAsAfkFilter: string;
```

Comment:
I think `always_active_pattern`, `filter_always_active`, `filter_assume_active` or something similar is a better name (5 word names is a bit much).

Reply:
modified the 5 word'ed var name. Done.